src/hammer-tech/hammer_tech.py

```diff
@@ -5,19 +5,28 @@
 #
 #  Copyright 2017-2018 Edward Wang <edward.c.wang@compdigitec.com>
 
-from abc import ABCMeta, abstractmethod
 import json
 import os
 import subprocess
-from typing import Any, Callable, List, NamedTuple, Optional, Union
+from abc import ABCMeta, abstractmethod
+from functools import reduce
+from numbers import Number
+from typing import Any, Callable, Iterable, List, NamedTuple, Optional, Union
 
 import hammer_config
-
-from hammer_logging import HammerVLSILoggingContext
-
 import python_jsonschema_objects  # type: ignore
-from hammer_utils import deeplist, get_or_else, optional_map
 from hammer_config import load_yaml
+from hammer_logging import HammerVLSILoggingContext
+from hammer_utils import (LEFUtils, add_lists, deeplist, get_or_else,
```

Comment:
Is there some style guide at which point you import * or is that not possible in python?

Reply:
IIRC the general rule is to avoid wildcard imports unless the package is dead simple e.g. has a bunch of constants.